README_template.md

```diff
@@ -103,6 +103,10 @@ Contributions are what make the open source community such an amazing place to l
   1. Submit Your xRunBook to the repository Follow the [submission steps](https://docs.unskript.com/unskript-product-documentation/guides/contribute-to-open-source) to remove credentials, etc. from your xRunBook.
   2. Submit a PR!
 
+## Import a xRunBook
+  1. xRunBooks are stored locally at ```/Users/<username/.unskript/runbooks``` . Place your existing RunBook in this directory.
```

Comment:
I think instead of `/Users/<username` We can say $HOME/.unskript. $HOME is a standard unix/mac/linux variable that is always set.

Reply:
done